engine/class_modules/paladin/sc_paladin_retribution.cpp

```diff
@@ -324,7 +324,14 @@ struct blade_of_justice_t : public paladin_melee_attack_t
 
       if ( p->talents.jurisdiction->ok() )
       {
-        base_multiplier *= 1.0 + p->talents.jurisdiction->effectN( 4 ).percent();
+        // Jurisdiction doesn't increase Expurgation's damage in-game
+        // base_multiplier *= 1.0 + p->talents.jurisdiction->effectN( 4 ).percent();
```

Comment:
Interesting; it's supposed to according to the spell data. Can you gate this behind a `bugs` check?

Reply:
Spell data increases direct damage, but Expurgation has no direct damage, only ticking damage. Jurisdiction increases Spell Direct Amount instead of Spell Periodic Amount (See Avenging Wrath effect 2 (and 11))
Sure, can do that

Reply:
Bugs-Flag added in https://github.com/simulationcraft/simc/pull/8169/commits/5bf5ff3a2ab6555cc25622a5b07ad4805379ea3e